src/nvim/eval.c

```diff
@@ -562,6 +570,14 @@ void eval_init(void)
   set_vim_var_list(VV_ERRORS, list_alloc());
   set_vim_var_nr(VV_SEARCHFORWARD, 1L);
   set_vim_var_nr(VV_HLSEARCH, 1L);
+  set_vim_var_nr(VV_TYPE_NUMBER, 0);
```

Comment:
Sadly I can't remember why I hard-coded them..but it had something to do with the fact that I got some compile errors trying to port it verbatim.